<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>441</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="quit_b">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>210</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Papyrus</family>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border-image: url(:/pics/button.png);
border-width: 12px;
padding: -12px 0px;
min-height: 25px;
min-width: 60px;
border:2px groove gray;
border-radius:10px;
}
QPushButton:pressed {
color: lightgray;
border-image: url(:/pics/button-pressed.png); 
padding-top: -10px;
padding-bottom: -16px;
}
QPushButton:hover {
border-width: 12px;
padding: -12px 0px;
min-height: 25px;
min-width: 60px;
border:3px groove gray;
border-radius:10px;
}</string>
     </property>
     <property name="text">
      <string>QUIT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_b">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>150</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Papyrus</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border-image: url(:/pics/button.png);
border-width: 12px;
padding: -12px 0px;
min-height: 25px;
min-width: 60px;
border:2px groove gray;
border-radius:10px;
}
QPushButton:pressed {
color: lightgray;
border-image: url(:/pics/button-pressed.png); 
padding-top: -10px;
padding-bottom: -16px;
}
QPushButton:hover {
border-width: 12px;
padding: -12px 0px;
min-height: 25px;
min-width: 60px;
border:3px groove gray;
border-radius:10px;
}</string>
     </property>
     <property name="text">
      <string>START GAME</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>381</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Papyrus</family>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; color:#ffffff;&quot;&gt;Catch Me if You Can&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; color:#ffffff;&quot;&gt;v0.1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="back_b">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>360</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Papyrus</family>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border-image: url(:/pics/button.png);
border-width: 12px;
padding: -12px 0px;
min-height: 25px;
min-width: 60px;
border:2px groove gray;
border-radius:10px;
}
QPushButton:pressed {
color: lightgray;
border-image: url(:/pics/button-pressed.png); 
padding-top: -10px;
padding-bottom: -16px;
}
QPushButton:hover {
border-width: 12px;
padding: -12px 0px;
min-height: 25px;
min-width: 60px;
border:3px groove gray;
border-radius:10px;
}</string>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>301</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gl"/>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
